bitcoin/src/psbt/mod.rs

```diff
@@ -553,6 +554,38 @@ impl Psbt {
     }
 }
 
+impl AsTransaction for Psbt {
```

Comment:
This is going to play really nicely with my new PSBTv2 stuff.

Reply:
Yeah, you could also have `struct CheckedPsbt<T: Borrow<Psbt>>(Psbt);` which guarantees the `Psbt` inside has all prevouts filled and implement `AsExtendedTransaction` on it. I didn't want to add it to this PR to not make it crazy big.